test/integration/integration_test.go

```diff
@@ -19,94 +19,27 @@ limitations under the License.
 package integration
 
 import (
-	"crypto/tls"
-	"encoding/json"
-	"errors"
-	"flag"
-	"fmt"
-	"io"
-	"io/ioutil"
-	"net"
-	"net/http"
-	"net/url"
 	"os"
-	"path/filepath"
-	"regexp"
-	"strconv"
 	"strings"
 	"testing"
-	"time"
 
 	"github.com/DATA-DOG/godog"
-	"github.com/DATA-DOG/godog/gherkin"
-	"gopkg.in/yaml.v2"
-
-	"github.com/minishift/minishift/pkg/minikube/constants"
-	testProxy "github.com/minishift/minishift/test/integration/proxy"
-	"github.com/minishift/minishift/test/integration/util"
-)
-
-var (
-	minishift       *Minishift
-	minishiftArgs   string
-	minishiftBinary string
-
-	testDir       string
-	testResultDir string
-	isoName       string
-
-	runBeforeFeature string
-	testWithShell    string
-	copyOcFrom       string
-
-	// Godog options
-	godogFormat              string
-	godogTags                string
-	godogShowStepDefinitions bool
-	godogStopOnFailure       bool
-	godogNoColors            bool
-	godogPaths               string
+	"github.com/minishift/minishift/test/integration/testsuite"
```

Comment:
I think we need a better name for this. I can not come up with any better name. @minishift/minishift-dev ?

Reply:
@LalatenduMohanty Agree, it is not a strong name. Trying to follow Go advices to keep it short this was the best I realized, but there might be better, would be happy to find it :)

Reply:
I actually think it is an OK name